content/en/account_management/plan_and_usage/cost_details.md

```diff
@@ -46,17 +45,18 @@ The cost summary functionality changes according to your Datadog usage as a sing
 
 {{< img src="account_management/plan_and_usage/multiorg-current-month-historical-costs.png" alt="Screenshot of the current month's Cost Summary for a parent organization, showing the overall month-to-date cost, projected cost, a graph with cumulative cost breakdowns, and a summary table including month-over-month cost changes." >}}
 
-View historical costs by toggling to previous months.
+View historical costs by toggling back to previous months or use the date dropdown to view costs over 1,3, 6 or 12 months.
 
-{{< img src="account_management/plan_and_usage/multiorg-prior-month-historical-costs.png" alt="Screenshot of a previous month's Cost Summary for a parent organization, showing the overall cost for the month, a graph with cumulative cost breakdowns, and a summary table including month-over-month cost changes." >}}
+{{< img src="static/images/account_management/plan_and_usage/parent-org-multi-month-cost-changes.png" alt="Screenshot of a parent organization's historical costs over a three month period, showing the overall cost for the month, a graph with cumulative cost breakdowns, and a summary table including month-over-month cost changes." >}}
```

Comment:
```suggestion
{{< img src="account_management/plan_and_usage/parent-org-multi-month-cost-changes.png" alt="Screenshot of a parent organization's historical costs over a three month period, showing the overall cost for the month, a graph with cumulative cost breakdowns, and a summary table including month-over-month cost changes." >}}
```

This image isn't rendering on your staging preview, we actually don't need to preface it with `/static/images`